src/main/java/basePackage/baseclass.java

```diff
@@ -47,7 +47,7 @@ else if (browserName.equals("IE"))
 //		IE code
 	}
 
-	driver.manage().timeouts().implicitlyWait(2, TimeUnit.SECONDS);
+	driver.manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS);
```

Comment:
Create a reusable function for implicit wait

Reply:
Working on it